### PR TITLE
[NGS-1066] Added request for iOS vs Android

### PR DESCRIPTION
**Pull request by Rajat2809**
With this change there will be 8 Request per auction request.

Jira: https://tapjoy.atlassian.net/browse/NGS-1066

### CODE REVIEW

File: exchange/bidder.go [hunk 1]
```diff
@@ -183,8 +184,13 @@ func (bidder *bidderAdapter) requestBid(ctx context.Context, request *openrtb2.B
 	if len(reqData) == 1 {
 		responseChannel <- bidder.doRequest(ctx, reqData[0])
 	} else {
+
 		for _, oneReqData := range reqData {
+			// get transaction for new relic
+			txn := newrelic.FromContext(ctx)
 			go func(data *adapters.RequestData) {
+				// add transaction to ctx to report multiple requests individually
+				ctx = newrelic.NewContext(ctx, txn)
```
Comment (umutdemirel) [inline exchange/bidder.go:193]: This won't work, you need to pass transaction in your go func, the value should be `txn.NewGoroutine()`

Just have a look at here: https://github.com/Tapjoy/tpe_prebid_service/blob/90805353ed9a3fb4a230497a592b965181ddf2f4/exchange/exchange.go#L497-L558

**Review by umutdemirel — CHANGES_REQUESTED** [review]
Experiment changes looks good, NR changes need an update

File: exchange/bidder.go (hunk 1 above)
Comment (umutdemirel) [reply exchange/bidder.go:193]: - Extract txn from your context (you did correct)
- Pass to your go func with `txn.NewGoroutine()`
- Use the go func param txn in your goroutine

Comment (Rajat2809) [reply exchange/bidder.go]: Did the change.

**Review by umutdemirel — APPROVED** [review]
Let's try it and see if it's gonna increase the number of calls. LGTM

**Pull request merged**